lib/estella/searchable.rb

```diff
@@ -59,7 +59,7 @@ module ClassMethods
       # indexes slug attribute by default
       def index_slug
         if defined? slug
-          indexed_fields.merge!(slug: { type: :string, index: :not_analyzed })
+          indexed_fields.merge!(slug: { type: :text, index: :not_analyzed })
```

Comment:
Is `keyword` more appropriate here, since it's not analyzed?

Reply:
In addition I think the doc at the top of this file can be updated to drop `string`.

Reply:
Yeah not sure if we use `:keyword` is that implicitly `:not_analyzed`?  Perhaps we can drop that parameter